app/questionnaire/router.py

```diff
@@ -82,22 +91,26 @@ def can_access_location(self, location: Location, routing_path):
 
         return location.block_id in self._get_allowable_path(routing_path)
 
-    def can_access_hub(self):
+    def can_access_hub(self) -> bool:
         return self._schema.is_flow_hub and all(
             self._progress_store.is_section_complete(section_id)
             for section_id in self._schema.get_section_ids_required_for_hub()
             if section_id in self.enabled_section_ids
         )
 
-    def can_display_section_summary(self, section_id, list_item_id=None):
+    def can_display_section_summary(
+        self, section_id: str, list_item_id: str = None
+    ) -> Union[ImmutableDict, None, bool]:
         return self._schema.get_summary_for_section(
             section_id
         ) and self._progress_store.is_section_complete(section_id, list_item_id)
 
-    def routing_path(self, section_id, list_item_id=None):
+    def routing_path(self, section_id: str, list_item_id: str = None) -> RoutingPath:
         return self._path_finder.routing_path(section_id, list_item_id)
 
-    def get_next_location_url(self, location, routing_path, return_to=None):
+    def get_next_location_url(
+        self, location: Location, routing_path: RoutingPath, return_to: str = None
```

Comment:
Same here for `return_to`, we should use `OptionL[str]`.

Reply:
Added, thanks!